science/gildas/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           compilers 1.0
 PortGroup           active_variants 1.1 
 
 name                gildas
-version             201904a
+version             201905a
 revision            1
```

Comment:
The `revision` needs to be reset to 0.

Reply:
I have removed the 'revision' in the following commit. Is this enough to accept the PR?

Reply:
Yes, now you just need to squash the commits into 1.

Reply:
Can I do this with the GitHub interface?

Reply:
Run `git rebase -i HEAD~2`, then change one of the `pick`s to `squash` and save the open file. Fix the commit message that comes up, and then `git push --force`

Reply:
Ok thanks. I was just wondering what happens in GitHub when I overwrite the branch (git push --force). This is done.